# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Define the action result message
Point2D[] merged_points
float32 radius_used